cmd/limactl/copy.go

```diff
@@ -33,6 +33,7 @@ func newCopyCommand() *cobra.Command {
 	}
 
 	copyCommand.Flags().BoolP("recursive", "r", false, "copy directories recursively")
+	copyCommand.Flags().BoolP("quiet", "q", false, "suppress output logs")
```

Comment:
I would consider making this quiet by default and adding -v --verbose for the verbose output, like the underling command we run and most commands.

Reply:
SGTM

Reply:
I believe that's what he meant by "sounds good to me" (SGTM).

Reply:
From the documentation, I can see that --debug flag is inherited. How will the introduction of --verbose flag affect the behaviour of the inherited flag or what distinct behaviour should the both have? @nirs